dicesGame/src/main/java/com/example/dices/repositories/IDiceRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.example.dices.repositories;
+
+import com.example.dices.models.DiceModel;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface IDiceRepository extends JpaRepository<DiceModel, Integer> {
+
+
```

Comment:
Please remove these two blank lines.

Reply:
@Repository
public interface IDiceRepository extends JpaRepository<DiceModel, Integer> {
}

This lines were deleted.

Reply:
No, the diff is still showing these lines. Did you commit and push the deletion of these lines?